recover-id-object/scripts/Dockerfile

```diff
@@ -16,7 +16,7 @@ COPY --from=build /usr/app/recover-id-object/target/release/recover-id-object ./
 
 RUN apt-get update && \
     apt-get -y install \
-      ca-certificates \
+      ca-certificates curl \
```

Comment:
Please document why this is installed.

Otherwise it's liable to be removed at some point during cleanup.